lib/groups/help.js

```diff
@@ -3,13 +3,18 @@ const chalk = require('chalk');
 const commandLineUsage = require('command-line-usage');
 
 class HelpGroup {
-    outputHelp(command = null) {
-        if (command !== null) {
-            const commands = require('../utils/commands').info;
+    outputHelp(subject = null, isCommand = true) {
```

Comment:
You should revert the properties. Usually the ones that are not needed (the ones that are `undefined` or `null`) should go after the one that have mandatory values.

Also, are you use you want to have a `null`? I mean, if you don't pass anything `isCommand` will be `undefined` which is good for this case, right?